source/_includes/asides/lovelace_navigation.html

```diff
@@ -1,6 +1,7 @@
 <section class="aside-module grid__item one-whole lap-one-half">
   <!-- {% include edit_github.html %} -->
   {% assign cards = site.lovelace | sort_natural: 'title' %}
+  {% assign cards = site.lovelace.badges | sort_natural: 'title' %}
```

Comment:
This now overwrites the line above...
Resulting in losing all the Lovelace cards? 😒

Reply:
Opps, didn't even mean to have this 🤦‍♂